frontend/apps/app/next.config.ts

```diff
@@ -30,6 +30,7 @@ const nextConfig: NextConfig = {
       },
     ],
   },
+  serverExternalPackages: ['@electric-sql/pglite'],
```

Comment:
Only this line was needed 🙏  . 

So this pr also reverts https://github.com/liam-hq/liam/pull/2970 🙏